client/src/components/ConversationModal.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { Comment, Grid, Modal, Button, Divider, Form, Icon} from 'semantic-ui-react'
+import { makeCall } from '../apis';
+
+/*
+ * DETAILS: Child component, conversation modal
+ * Shows full conversation history and allows for sending of messages
+ * 
+ * PROPS: userDetails,
+ *        conversation {messages [{message, sender, datesent, dateString}], 
+ *                      alumni[{name, imageURL}], seen[bool], dateCreated }
+ *        modalOpen
+ *        closeModal()
+ */ 
+export default class Conversation extends Component {
+    constructor(props){
+        super(props)
+        this.state={
+            conversation: null,
+            display: [],
+            message: ''
+        }
+        this.handleValueChange = this.handleValueChange.bind(this)
+        this.sendMessage = this.sendMessage.bind(this)
+    }
+    componentWillMount() {
+        this.createDisplay(this.props.conversation)
+        this.setState({conversation: this.props.conversation})
+    }
+
+    async sendMessage() {
+        let conversation = await makeCall(
```

Comment:
nit: can we call this result, so we can access the conversation using `result.conversation`?